testCPUID-all/testCPUID.cpp

```diff
@@ -371,13 +371,48 @@ void parse_cmd_args(int argc, char** argv)
 void SetThreadAffinity(HANDLE tHandle, int procNum)
 {
     GROUP_AFFINITY ga;
-    //ga.Group = (WORD)groupProcNo.GetGroup();
     ga.Group = (WORD)(procNum >> 6);
     ga.Reserved[0] = 0; // reserve must be filled with zero
     ga.Reserved[1] = 0; // otherwise call may fail
     ga.Reserved[2] = 0;
-    //ga.Mask = (size_t)1 << groupProcNo.GetProcIndex();
     ga.Mask = (size_t)1 << (procNum % 64);
+
```

Comment:
can you please make a note how this mapping is obtained? by trial and error or was there some more systematic way?

Reply:
@Maoni0 - added. Let me know if it makes sense.